### ISSUE TITLE
[BUG] New TnOptions not yet supported: NNID, ESPID, A2pSettings, Origination Route Plan, Pindrop

### ISSUE THREAD

**Issue by ooglek** [Bandwidth/php-bandwidth-iris#91]
### Description
New TN Option values available in the `CreateTnoptionOrder` API are not yet supported in the SDK.

https://dev.bandwidth.com/apis/numbers/#operation/CreateTnoptionOrder

- NNID [ integer | unchanged | systemDefault ]
- ESPID [ string | unchanged | systemDefault ]
- A2pSettings
- Origination Route Plan
- Pindrop Enabled [ string | true | false ]

### Environment Information

 - OS Version: FreeBSD
 - SDK Version: v4.0.1
 - Environment: PHP 7.3

### Suggested Fix

I'll submit a Pull Request shortly, though I might only implement NNID and ESPID as the rest are likely a bit more complex.

**Comment by ooglek** [comment]
Also it seems like you cannot change the NNID for a single TN. That's annoying, that's precisely why we want to use this!!!

I'll use the "bulk update" in the Dashboard for now. Basically for the number I'm working on, SMS stopped working, likely an NNID change that was not authorized but it was done anyway.

```
>>> $a = $bw->account->tnoptions();
>>> $b = $a->create(['TnOptionGroups' => ['TnOptionGroup' => ['NNID' => '123456', 'TelephoneNumbers' => ['TelephoneNumber'=>'2345678900']]]]);
Iris\ResponseException with message 'NNIDs for Bandwidth On-net TNs TNs are not configured for per-TN NNID update'
```

**Comment by ooglek** [comment]
Thanks @ajrice6713 ! Email sent.

**Comment by ooglek** [comment]
Current diff, no PR yet:
```
Index: include/vendor/bandwidth/iris/src/simpleModels/TnOptionGroups.php
===================================================================
--- include/vendor/bandwidth/iris/src/simpleModels/TnOptionGroups.php   (revision 6921)
+++ include/vendor/bandwidth/iris/src/simpleModels/TnOptionGroups.php   (working copy)
@@ -26,7 +26,9 @@
         "Protected" => array("type" => "string"),
         "Sms" => array("type" => "string"),
         "FinalDestinationURI " => array("type" => "string"),
-        "PortOutPasscode" => array("type" => "string")
+        "PortOutPasscode" => array("type" => "string"),
+        "NNID" => array("type" => "string"),
+        "ESPID" => array("type" => "string")
     );
     public function __construct($data) {
         $this->set_data($data);
```

**Comment by imanimanyara** [comment]
> Current diff, no PR yet:
> 
> ```
> Index: include/vendor/bandwidth/iris/src/simpleModels/TnOptionGroups.php
> ===================================================================
> --- include/vendor/bandwidth/iris/src/simpleModels/TnOptionGroups.php   (revision 6921)
> +++ include/vendor/bandwidth/iris/src/simpleModels/TnOptionGroups.php   (working copy)
> @@ -26,7 +26,9 @@
>          "Protected" => array("type" => "string"),
>          "Sms" => array("type" => "string"),
>          "FinalDestinationURI " => array("type" => "string"),
> -        "PortOutPasscode" => array("type" => "string")
> +        "PortOutPasscode" => array("type" => "string"),
> +        "NNID" => array("type" => "string"),
> +        "ESPID" => array("type" => "string")
>      );
>      public function __construct($data) {
>          $this->set_data($data);
> ```

Thanks @ooglek for pointing this out. Has it been fixed? also facing same issue.

**Comment by ajrice6713** [comment]
Fixed in v4.1.0!
